src/VocabGameData.js

```diff
@@ -0,0 +1,39 @@
+const basePairs = [
+  ["hello", "bonjour"],
+  ["goodbye", "au revoir"],
+  ["please", "s'il vous plaît"],
+  ["thank you", "merci"],
+  ["thank you very much", "merci beaucoup"],
+  ["you're welcome", "de rien"],
+  ["excuse me", "excusez-moi"],
+  ["yes", "oui"],
+  ["no", "non"],
+  ["can you help me, please?", "pouvez-vous m’aider, s’il vous plaît?"]
+];
+
+let dataset = [];
+function createDataset(base) {
+  base.forEach(function(array) {
+    dataset.push([
+      array[0],
```

Comment:
are you trying to use pure arrays, or a MAP object here?

Reply:
oh i see, you are making it an array first and then converting to a MAP object at the end.

Reply:
Yes, but this isn't any particular decision I made, it's because I didn't know if I could do it directly with MAP